src/containers/MarketListPage/index.js

```diff
@@ -9,6 +9,8 @@ import { getDefaultAccount } from 'selectors/blockchain'
 
 import { requestMarkets } from 'actions/market'
 
+const config = require('config.json')
```

Comment:
why not import?

Reply:
config.json contains a JSON dictionary, not a Javascript object. Is there a way to use Import with json files?

Reply:
What happens if you try `import config from 'config.json'`?